src/ui/characterinfoview.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module special-characters/ui/characterinfoview
+ */
+
+import View from '@ckeditor/ckeditor5-ui/src/view';
+
+import '../../theme/characterinfo.css';
+
+export default class CharacterInfoView extends View {
+	constructor( locale ) {
+		super( locale );
+
+		const bind = this.bindTemplate;
+
+		this.set( 'character', null );
+		this.set( 'name', null );
+
+		this.bind( 'code' ).to( this, 'character', char => {
```

Comment:
Not so needed but maybe extract this to `charcterToUnicode` function as implementation is not needed here - only what happens in this binding.